apps/mobile/src/features/Assets/components/Fallback/Fallback.tsx

```diff
@@ -1,25 +1,10 @@
 import React from 'react'
 import { getTokenValue, Spinner } from 'tamagui'
 
-import { Alert } from '@/src/components/Alert'
 import { SafeTab } from '@/src/components/SafeTab'
 
-export const Fallback = ({
-  loading,
-  hasError,
-  children,
-}: {
-  loading: boolean
-  hasError: boolean
-  children: React.ReactElement
-}) => (
+export const Fallback = ({ loading, children }: { loading: boolean; children: React.ReactElement }) => (
   <SafeTab.ScrollView style={{ padding: getTokenValue('$4') }}>
-    {loading ? (
-      <Spinner size="small" />
-    ) : hasError ? (
-      <Alert type="error" message={`Error fetching assets list`} />
-    ) : (
-      children
-    )}
+    {loading ? <Spinner size="small" /> : children}
```

Comment:
Let's craete our custom Spinner component and use CirclSnail from react-native-progress. We already use it in multiple components.